### ISSUE TITLE
Is this repo still alive?

### ISSUE THREAD

**Issue by Braintelligence** [dokuweaki/dokuweaki.github.io#1]
Hi everyone!
I found this repo by looking for a way to have real-time collaborative editing in DokuWiki. I'm sad to see that the commit-activity is very scarce.

Is it planned to continue development or does anyone know of an alternative for my use-case?

**Comment by ferrolho** [comment]
@Braintelligence, this was originally motivated for a University course project. None of the members involved seems to have had the time to continue development.

I am not aware of any alternatives to real-time collaboration in DokuWiki since I have stopped using it ever since. But is the current codebase not enough for your use-case?

**Comment by Braintelligence** [comment]
The current codebase is not enough to justify using it as a mandatory part of a long-lasting production environment, as long as further development is not at least to be expected, I'm afraid. 

Thanks for the quick reply, though and I hope you guys got a good grade!

EDIT: Wow, there was even no reaction on stackoverflow to your work: https://stackoverflow.com/questions/39441474/cross-domain-requests-with-sharejs
I'm really baffled this didn't gain traction... I really don't understand.

**Comment by ferrolho** [comment]
@Braintelligence, yes I was also surprised at the time with the lack of interest on this...

Anyway, this course is quite open to suggestions from the community for future projects. So if you are interested and have a substantial request that can match both your interests and those of the course organisers, I would suggest you to ping Prof. @aaguiar.

**Comment by Braintelligence** [comment]
@ferrolho I'll look into it. If I can dig myself into the codebase and can't find any alternative, I might follow up on your suggestion.

**Comment by Braintelligence** [comment]
It looks like you can combine https://github.com/michael-dev/dokuwikietherpadlite with http://etherpad.org/

I will look into this combination instead for now.
